tests/integration/fixtures/__init__.py

```diff
@@ -423,6 +423,32 @@ def send_unreal_request(self, project_id, file_content, dsn_key_idx=0):
         response.raise_for_status()
         return response
 
+    def send_playstation_request(self, project_id, file_content, dsn_key_idx=0):
+        """
+        Sends a request to the playstation endpoint
+        :param project_id: the project id
+        :param file_content: the unreal file content
+        """
+        response = self.post(
+            "/api/{}/playstation/?sentry_key={}".format(
+                project_id, self.get_dsn_public_key(project_id, dsn_key_idx)
+            ),
+            headers={
+                "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+                "(KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36",
+            },
```

Comment:
Is that a real user agent from a PS?

Reply:
No it is a copy of the user agent for the `unreal_request` in the function above but since the tests seem to work without this header anyways might as well remove it 🤔